dbt/adapters/databricks/python_submissions.py

```diff
@@ -146,7 +154,7 @@ def _submit_through_notebook(self, compiled_code: str, cluster_spec: dict) -> No
                 "Python model failed with traceback as:\n"
                 "(Note that the line number here does not "
                 "match the line number in your code due to dbt templating)\n"
-                f"{json_run_output['error_trace']}"
+                f"{utils.remove_ansi(json_run_output['error_trace'])}"
```

Comment:
Most of the changes to this file are just VS Code autoformatting.  This line and 394 are the only real changes.